recipes/cccl-python/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "cccl-python" %}
+{% set version = "3.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NVIDIA/cccl/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 3ded63682f94071685cd6492cee70ad2a1feae02abfb3fbcce50e2d2f22edf4e
+
+build:
+  script: |
+    cd python/cuda_cccl
+    {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
```

I don't think stage-recipes has CUDA 13 ready for use? Let's build it with CUDA 12 for now, and turn on CUDA 13 support after merge & a feedstock is created.
```suggestion
  number: 0
  string: cuda{{ cuda_compiler_version | replace('.', '') }}_py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [cuda_compiler_version != "None"]
  skip: true  # [(py < 39) or cuda_compiler_version not in ("12.9",) or ppc64le]
  ignore_run_exports_from:
    - {{ compiler('cuda') }}
```